src/FeatureExtractor/Mobilenet.js

```diff
@@ -121,7 +121,7 @@ class Mobilenet {
   async addImageInternal(imgToAdd, label) {
     await this.ready;
     tf.tidy(() => {
-      const processedImg = imgToTensor(imgToAdd);
+      const processedImg = imgToTensor(imgToAdd, [224, 224]);
```

Comment:
If a video is passed, the video gets resized when the method loads. Calling this again with a video will make it resize again (See https://github.com/ml5js/ml5-library/blob/cb8651e477d0f307ecf8bc7a3f6e650ac78a3c2e/src/utils/imageUtilities.js#L68-L70)

We should check if there's a video and call `imgToTensor` with out size in that case.